src/interfaces/Registration/business-lookup-interface.ts

```diff
@@ -0,0 +1,19 @@
+export interface BusinessLookupIF {
+  identifier: string
+  name: string
+  bn?: string
```

Comment:
Is it truly optional? You assign it in EmptyBusinessLookup so maybe this isn't needed?